### PR TITLE
No space parsing for ever_given output parsing

### PR DESCRIPTION
**Pull request by megosato**

### CODE REVIEW

**Comment by megosato** [comment]
I haven't updated the package yet, was going to wait on Braxton before updating pypi

**Comment by mikemhenry** [comment]
@megosato It should be safe to update the package since we have things pinned + we can always remove a package from pypi if it is broken. But I do think we have something weird going on, since we should get an error that we can't find that version of `ever_given`...

File: app/core/templates/core/submission_list.html [hunk 1]
```diff
@@ -2,9 +2,11 @@
 {% block content %}
 <h1>Your submissions</h1>
 <table class="table table-striped">
+<!--- <table class="table table-striped"> --->
```
Comment (robbason) [inline app/core/templates/core/submission_list.html:5]: why?

**Review by robbason — CHANGES_REQUESTED** [review]
just a little cleanup, looks good otherwise! Please let's squash and merge to keep commit history clean. lmk if you have questions!

**Review by robbason — APPROVED** [review]

**Pull request merged**